.chglog/CHANGELOG.tpl.md

```diff
@@ -0,0 +1,111 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+{{ if .Versions -}}
+<a name="unreleased"></a>
```

Comment:
why the `<a` tag is needed ?

Reply:
I am not sure, I took this from the cq-terraform repo. I can't test it until the pr is merged because the gh actions won't run